content/docs/studio/user-guide/change-team-plan-and-size.md

```diff
@@ -0,0 +1,111 @@
+
+In this section, you learn how to:
+
+- [Change team plan](#change-team-plan)
+- [Change team size](#change-team-size)
+
+## Change Team Plan
+
+When you create a team, it is on the Free plan by default. This plan is designed
+for teams just starting out with collaboration: you can invite one other team
+member and import as many repositories as you want. For more advanced
+collaboration functionality, you can upgrade to the Team or Enterprise plans. A
+detailed comparison of the different plans can be found in the Iterative Studio
+[pricing page](https://studio.iterative.ai/pricing).
+
+To change the plan for your team, go to the `Plan and billing` section within
+Team settings. Then, within the `Current plan` card, click on `Change plan`.
+
+![](https://static.iterative.ai/img/studio/team_settings_plan_and_billing.png)
+
+A plan comparison page will open. This page provides detailed comparison of the
+different plans in Iterative Studio. Your current plan will be marked with a
+`Current plan` label.
+
+![](https://static.iterative.ai/img/studio/team_settings_plans_and_features.png)
+
+### If you are currently on the Free plan
+
+You can upgrade to the Team plan or to the Enterprise plan.
+
+- **To upgrade to the Team plan**, click on the `Upgrade` button. Then, select
+  the number of seats (collaborators) you want in your team.
+
+![](https://static.iterative.ai/img/studio/team_settings_select_seats.png)
+
+The total payable amount will be shown. For details of how the payable amount is
+calculated, refer to
+[billing and payment](/doc/studio/user-guide/billing-and-payment).
+
+To make the payment and upgrade the plan, click on `Confirm`. You will be
+redirected to our payment processor, [Stripe](https://stripe.com/), for making
+the payment. Once your payment is successful, you will be redirected back to the
+team settings page in Iterative Studio. Here, you will see that your team is now
+in the Team plan.
+
+<admon>
+
+Each collaborator and pending invite takes up a seat in the team. For example,
+if you have 5 seats in your team which has 3 collaborators and 2 pending
+invites, you cannot add any more members until you remove a collaborator or
+pending invite, or increase the number of seats.
+
+</admon>
+
+- **To upgrade to the Enterprise plan**, click on the `Contact Us` button and
+  you will be redirected to a Calendly form where you can schedule a call with
+  our in-house experts who will try to understand your needs and suggest a
+  suitable plan and pricing.
+
+### If you are currently on the Team plan
+
+You can downgrade to the Free plan or upgrade to the Enterprise plan.
+
+- **To downgrade to the Free plan**, you must first make sure that your team
+  does not have more than 2 collaborators or pending invites. Then, click on the
+  `Downgrade` button.
+
+  When you downgrade, you will receive a refund based on the number of days
+  remaining in the billing period. If there are any discrepancies in the
```

Comment:
If discrepancies are not expected, I think mentioning them is rather alarming. We can replace these with "for any inquiries about refunds, you can contact us."

Reply:
Changed.
We do expect some issues during refunds, if a team has a positive account balance at the time that they cancel their subscription. So, we needed to add this.